Tech Radar 2019.md

```diff
@@ -0,0 +1,39 @@
+At all times:
+
+- browser and hardware interoperation
+- accessibility
+- responsive and progressive content
+- longevity of content
+- user-centric privacy and security
+
+2019:
+
+- WebXR Device API
+- content creation tools like Blender integration of glTF
+- frameworks like Sumerian, PotassiumES
+
+2020:
+
+- payment integration
+- hand gesture input
+- eye tracking
+
+2021:
+
+- shared anchors
+- long lived, simultanous XR apps
+- deeper OS integration
+
+2022:
+
+- shared layers / spaces
+- delarative content
+- personal agents
+- page-defined computer vision
+- portable avatars
```

Comment:
Without context, this is "meaningless" I think.  Avatars for what?  How are they rendered?  How are they included in a page / app?  Or are they supported by the browser?  We could do a "gravatar3d" tomorrow, but who would use it and how?

Reply:
Yeah, I'm doing something like this with Simbol (library for content creators to use), but that's far from WebXR native browser extensions or proper browser implementation.